crowdin.dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM openjdk:11
+# copied from https://github.com/meisterplan/docker-crowdin-cli/blob/master/Dockerfile
+# Doc from crowdin-cli https://support.crowdin.com/cli-tool/
+ENV CROWDIN_VERSION=2.0-22
+
+RUN apt-get update && \
+    apt-get install -y make git && \
+    apt-get clean
+RUN wget -qO - https://artifacts.crowdin.com/repo/GPG-KEY-crowdin | apt-key add -
+RUN echo "deb https://artifacts.crowdin.com/repo/deb/ /" > /etc/apt/sources.list.d/crowdin.list
+RUN apt-get update && \
+    apt-get install -y crowdin=${CROWDIN_VERSION} && \
```

Comment:
line 7 and this can be merged removing a layer from the container.
As well you should delete the apt-get lists after installing something and can avoid additional packages by specifying `--no-install-recommends` to have a smaller container